src/components/layouts/LayoutContainer.tsx

```diff
@@ -17,6 +17,7 @@ export const LayoutContainer = ({
     layoutProps,
     layoutStyle,
     children,
+    ...rest
```

Comment:
Litt usikker på hva _rest_ er, gitt at Props bare er pageProps, layoutProps, layoutStyle og children?

Reply:
Dette var en fiks for å få visse props gjennom designsystems komponenter tror jeg. Litt utydelig kanskje, har endret det nå.